config-linux.md

```diff
@@ -118,6 +118,21 @@ For more information, see [the man page](http://man7.org/linux/man-pages/man8/sy
 
 rlimits allow setting resource limits. The type is from the values defined in [the man page](http://man7.org/linux/man-pages/man2/setrlimit.2.html). The kernel enforces the soft limit for a resource while the hard limit acts as a ceiling for that value that could be set by an unprivileged process.
 
+## Linux user namespace mappings
+
+```
+    "userMappings": [
+        {
+            "from"  : 1000,
+            "to"    : 0,
+            "count" : 10
```

Comment:
That's how we have it today in the spec. We could either go with how it is in the kernel or what we have in golang.